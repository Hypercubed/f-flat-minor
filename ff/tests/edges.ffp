.import ../lib/core.ff
.import ../lib/testing.ffp

0 'Case\sInsensitive' prints
  x: 1 2 + ;
  X 3 assert_eq

  Y: 5 8 + ;
  y 13 assert_eq

  [Z] : 21 34 + ;
  z 55 assert_eq
OK

0 'Compiler\sCommands\n' prints
.this should be ignored
1 2 + .this_also_ignored 3 assert_eq
5
. 8 + /* this should not be ignored */
13 assert_eq
OK