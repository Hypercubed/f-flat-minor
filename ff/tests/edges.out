Case Insensitive OK
Compiler Commands
[ 5 ]
 OK
